src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/api/CardEventHandlerTest.java

```diff
@@ -284,4 +286,133 @@ void does_not_change_gamestate_if_its_not_the_last_card() {
     assertThat(updatedGame.getMatches(), hasSize(9));
     assertThat(updatedGame.getGameState(), is(PLAYING));
   }
+
+  @Test
+  void play_last_card_with_round_stacked() {
+    // this test may fail in case in the battling round multiple players have the same highest cards
+    // again
+    Game game =
+        GameBuilder.builder("test", gameRepository, participationRepository, playerRepository)
+            .withParticipation(PLAYER_NAME_1)
+            .withParticipation(PLAYER_NAME_2)
+            .withParticipation(PLAYER_NAME_3)
+            .withMatch()
+            .withHandForPlayer(PLAYER_NAME_1)
+            .withCards(ACE_OF_CLUBS, QUEEN_OF_CLUBS, JACK_OF_CLUBS)
+            .finishHand()
+            .withHandForPlayer(PLAYER_NAME_2)
+            .withCards(KING_OF_CLUBS, JACK_OF_SPADES, EIGHT_OF_CLUBS)
+            .finishHand()
+            .withHandForPlayer(PLAYER_NAME_3)
+            .withCards(SEVEN_OF_CLUBS, ACE_OF_SPADES, QUEEN_OF_SPADES)
+            .finishHand()
+            .withRound()
+            .withPlayedCard(PLAYER_NAME_1, JACK_OF_CLUBS)
+            .withPlayedCard(PLAYER_NAME_2, KING_OF_CLUBS)
+            .withPlayedCard(PLAYER_NAME_3, ACE_OF_SPADES)
+            .finishRound()
+            .withRound()
+            .withPlayedCard(PLAYER_NAME_1, ACE_OF_CLUBS)
+            .withPlayedCard(PLAYER_NAME_2, JACK_OF_SPADES)
+            .withPlayedCard(PLAYER_NAME_3, SEVEN_OF_CLUBS)
+            .finishRound()
+            .withRound()
+            .withPlayedCard(PLAYER_NAME_1, QUEEN_OF_CLUBS)
+            .withPlayedCard(PLAYER_NAME_2, EIGHT_OF_CLUBS)
+            .withPlayedCard(PLAYER_NAME_3, QUEEN_OF_SPADES)
+            .finishRound()
+            .finishMatch()
+            .build();
+
+    gameRepository.saveAll(List.of(game));
+    List<Hand> sortedHands = getHandsSortedByParticipation(game);
+    Round battlingRound = match.getLastRound().orElseThrow();
+    Hand handPlayer1 = sortedHands.get(0);
+    Hand handPlayer2 = sortedHands.get(1);
+    Hand handPlayer3 = sortedHands.get(2);
+
+    simulateBattlingCardPlaying(handPlayer1, battlingRound);
+    simulateBattlingCardPlaying(handPlayer2, battlingRound);
+    simulateBattlingCardPlaying(handPlayer3, battlingRound);
+
+    assertThat(handPlayer2.getNumberOfWonTricks(), is(0));
+    assertTrue(
+        handPlayer1.getNumberOfWonTricks() == 3 && handPlayer3.getNumberOfWonTricks() == 1
+            || handPlayer1.getNumberOfWonTricks() == 1 && handPlayer3.getNumberOfWonTricks() == 3);
+  }
+
+  @Test
+  void play_last_card_with_last_two_rounds_stacked() {
+    // this test may fail in case in the battling round multiple players have the same highest cards
+    // again
```

Comment:
```suggestion
```
Das merkst du selber, wenn man die history anschaut

Reply:
Was?

Reply:
Ich denke den kommentar braucht es nicht

Reply:
Nein, den braucht es nicht mehr. 
Mein Problem hier ist, dass ich meine, dieser Test müsste ab und zu failen, und zwar immer dann, wenn mehrere Spieler, welche um die letzte gestackte Runde kämpfen, dieselbe höchste Karte erhalten in der Zusatzrunde. Allerdings ist es noch nie vorgekommen und ich liess die Tests schon ziemlich oft laufen. Bei Gelegenheit würde ich das mal noch debuggen und schauen, was passiert, wenn ich das Szenario mit zwei gleichen höchsten Karten simuliere.